### ISSUE TITLE
Renaming of dimension semantic tokens

### ISSUE THREAD

**Issue by julien-deramond** [Orange-OpenSource/ouds-ios#130]
### Description

Dimension semantic tokens have been renamed. Their suffix was a number value (e.g. 0, 10, etc.), and is now a t-shirt size (e.g. Xs, Xl, etc.).

Here are the new values/matching coming straight from Style Dictionary (sorry, you'll have maybe to reorder them 😬):

```swift
public static let dimensionXs = dimension400
public static let dimensionXl = dimension600
public static let dimensionSm = dimension450
public static let dimensionNone = dimension0
public static let dimensionMd = dimension500
public static let dimensionLg = dimension550
public static let dimension9xs = dimension50
public static let dimension9xl = dimension1400
public static let dimension8xs = dimension75
public static let dimension8xl = dimension1200
public static let dimension7xs = dimension100
public static let dimension7xl = dimension1000
public static let dimension6xs = dimension150
public static let dimension6xl = dimension900
public static let dimension5xs = dimension200
public static let dimension5xl = dimension800
public static let dimension4xs = dimension250
public static let dimension4xl = dimension750
public static let dimension3xs = dimension300
public static let dimension3xl = dimension700
public static let dimension2xs = dimension350
public static let dimension2xl = dimension650
public static let dimension11xl = dimension1800
public static let dimension10xs = dimension25
public static let dimension10xl = dimension1600
```

**Comment by pylapp** [comment]
@julien-deramond We should apply Swift naming convention, thus the variable must have their suffix uppercased, e.g.:

```swift
public static let dimensionXS = dimension400
public static let dimensionXL = dimension600
public static let dimensionSM = dimension450
public static let dimensionNone = dimension0
public static let dimensionMD = dimension500
public static let dimensionLG = dimension550
public static let dimension9XS = dimension50
public static let dimension9XL = dimension1400
public static let dimension8XS = dimension75
public static let dimension8XL = dimension1200
public static let dimension7XS = dimension100
public static let dimension7XL = dimension1000
public static let dimension6XS = dimension150
public static let dimension6XL = dimension900
public static let dimension5XS = dimension200
public static let dimension5XL = dimension800
public static let dimension4XS = dimension250
public static let dimension4XL = dimension750
public static let dimension3XS = dimension300
public static let dimension3XL = dimension700
public static let dimension2XS = dimension350
public static let dimension2XL = dimension650
public static let dimension11XL = dimension1800
public static let dimension10XS = dimension25
public static let dimension10XL = dimension1600
```

**Comment by julien-deramond** [comment]
Yep, you can use the right format in your files for this modification, no problem. I copy-pasted the data I had from the parser as it was more convenient.

Regarding this use case, I'll make the parser smarter to adapt 👌 Added to my TODO.

**Comment by pylapp** [comment]
📣 **New TestFlight ALPHA uploap available** 🚀

Please, get the new build from *TestFlight* app.

- Display name: **OUDS Showcase ALPHA (117, 119, 122, 129, 130, 132)**
- Version: **0.3.0**
- Build number: **1727290249**
- Build tag: **37891af**
- Build type: **ALPHA**
- Build details (GitHub): **117, 119, 122, 129, 130, 132**

_This is an automated message sent with love and *Fastlane* from our CI/CD pipeline 🤘_

**Comment by pylapp** [comment]
New version available since version [v0.3.0](https://github.com/Orange-OpenSource/ouds-ios/releases/tag/0.3.0) (see [this thread](https://github.com/Orange-OpenSource/ouds-ios/discussions/166))
